requirements/docs.in

```diff
@@ -3,18 +3,22 @@
 -r pip.txt
 
 sphinx_rtd_theme==1.1.0
+# Note: Version 3.4.1 of sphinx-tabs requires docutils 0.18 which is yet to be supported by sphinx-rtd-theme
+# Version 3.4.0 has an incompatible Jinja2 version which also blocks sphinx-rtd-theme
+# All-together, we cannot upgrade to Sphinx 5.x before either sphinx-tabs or sphinx-rtd-theme fixes these
+# issues.
 sphinx-tabs==3.3.1
```

Comment:
We can probably migrate `sphinx-tabs` to `sphinx-design`: https://sphinx-design.readthedocs.io/en/latest/tabs.html at some point

Reply:
That would be really great. 1) to have one extension less and 2) I got an impression that sphinx-tabs is less maintained nowadays.